### DISCUSSION TITLE
please help> platform not working. hosted on cloud

### DISCUSSION THREAD

**Discussion by osanaji** [Budibase/budibase#4283] [Help]
Is anyone facing problems using app builder from the middle east? each time i opened the design view I found this error msg (App preview failed to load An unknown error occurred). even published apps not working with error msg (There was an error loading your app The Budibase client library could not be loaded. Try republishing your app.)

Note: I created a new account and created an app from existing thems and face the same problem.

**Comment by pavleks** [comment]
Hi @osanaji based on the previous [discussion ](https://github.com/Budibase/budibase/discussions/1979) I could imagine something related to the browser or its extensions. Could you try a different browser or maybe incognito mode (when extensions are disabled ) ?

**Reply by osanaji** [reply]
I think the problem is with budibase hosting service (free cloud account). I created a new account using a different machine and I still have the same problem.

**Reply by pavleks** [reply]
I am on budibase hosting service myself and everything works. Unfortunately, I cannot replicate the issue. Maybe two different machines using the same browser and sharing the same list of extensions? Previously the guy had Dashlane (password manager) after disabling it everything worked.

**Comment by osanaji** [comment]
Any help Please>

**Reply by pavleks** [reply]
Hi @osanaji 
Please could you share a bit more details? What web browser do you use (Chrome, Edge, Firefox, Safari) ?

**Reply by skotzko** [reply]
I was able to recreate this today. 

I published a tool, tested it on desktop Chrome, but when I try to load the tool on my iPhone w/ mobile safari, it fails with the "There was an error loading your app" screen.

**Reply by oboehlk** [reply]
The same here with an older iPad (12.4.1). Android 9 and Mac OS works fine.

**Comment by Ron-de** [comment]
Hi Everyone , 

I also got the same error There was an error loading your app" , I am using Chrome. 

Any workaround ?


![screen ](https://github.com/Budibase/budibase/assets/83381673/243f29a3-5baf-4236-9448-820fd32e1503)

**Reply by melohagan** [reply]
Hey @Ron-de Which version of Budibase are you running?

**Reply by Ron-de** [reply]
This is Cloud free tier, I just signed up and created an app.

**Reply by Ron-de** [reply]
Is there anyone who can help me with this ?

**Comment by Ron-de** [comment]
Is there anyone who can help me with this ?
